docs/source/standard.md

```diff
@@ -203,41 +203,100 @@ The ability of the TRE organisation to administer and secure network infrastruct
 
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
-
 #### 2.4.1 Deployment management
 
-The ability of the TRE organisation to instantiate, deploy, change or remove deployed infrastructure.
-
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+_The ability of the TRE organisation to instantiate, deploy, change or remove deployed infrastructure._
+
+```{list-table}
+:header-rows: 1
+:name: tab-deployment-management
+* - Statement
+  - Guidance
+  - Mandatory status
+* - You must have a documented procedure for deploying infrastructure.
+  - This might, for instance, be a handbook that is followed or a set of automated scripts.
+  - Mandatory
+* - Where possible, you should automate any repeatable aspects of your deployment.
+  - This might involve using infrastructure-as-code tools or simply a series of scripts.
+  - Recommended
+* - You must test changes before they are used in production.
+  - This might involve a separate development environment or another system for testing.
+  - Mandatory
+* - You could test changes in a development environment that mirrors your production system.
+  - Consider the costs and practicality of whether this will work for your situation.
+  - Optional
+* - You must have a documented procedure for making changes to deployed infrastructure.
+  - This refers both to changes that might be expected in the course of normal operation and emergency changes that might be needed.
+  - Mandatory
+* - You must have a documented procedure for removing infrastructure when it is no longer needed
+  -
+  - Mandatory
+```
 
 #### 2.4.2 Capacity management
 
-The ability of the TRE organisation to ensure the right amount of resources are available at the right time to provide a service.
-
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+_The ability of the TRE organisation to ensure the right amount of resources are available at the right time to provide a service._
+
+```{list-table}
+:header-rows: 1
+:name: tab-capacity-management
+* - You must ensure that all projects understand what resources are available and what the associated costs will be before the project starts.
+  - For on-premises systems this might be related to the available hardware, for cloud-based systems there might be limits on how many instances of a particular resource (_e.g._ GPUs) can be used. Projects should use this information to understand whether the available resources will be sufficient for their requirements.
+  - Mandatory
+* - You should ensure that the anticipated needs of projects can be satisfied using available resources.
+  - Note that this does not require you to accept requests for additional resources, but rather that promises made about resource availability before a project starts should be honoured wherever possible.
+  - Recommended
+* - You must ensure that sufficient budget is available to support the anticipated need for resources.
```

Comment:
Is this referring to the TRE's budget, or the project's budget in a pay-as-you-go scenario?